### PR TITLE
Lastfm UI

### PR DESCRIPTION
**Pull request by KaroukinStanislau**

### CODE REVIEW

File: lastFm/index.html [hunk 1]
```diff
@@ -5,14 +5,47 @@
...
 
 <body>
+
+  <div class="wrapper">
+    <header class="header">
+      <div class="container">
+        <div class="title">
+          <a class="no-link" href="">Last.fm API</a>
+        </div>
+        <div class="search">
```
Comment (NikitaSedyx) [inline lastFm/index.html:18]: is search available on all views?

File: lastFm/js/pagination.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+var Pagination = {
+
+  code: '',
+
+  Extend: function(data) {
```
Comment (NikitaSedyx) [inline lastFm/js/pagination.js:5]: `extend` etc

**Review by NikitaSedyx — COMMENTED** [review]
please remove console.logs

File: lastFm/index.html (hunk 1 above)
Comment (KaroukinStanislau) [reply lastFm/index.html:18]: yes. only need on topartist?

Comment (NikitaSedyx) [reply lastFm/index.html:18]: yes

**Pull request merged**